src/app/file-browser/components/file-list/file-list.component.ts

```diff
@@ -346,9 +346,11 @@ export class FileListComponent
 
 		const queryParams = this.route.snapshot.queryParamMap;
 		if (queryParams.has('showItem')) {
-			const folder_linkId = Number(queryParams.get('showItem'));
+			const folderLinkId = Number(queryParams.get('showItem'));
```

Comment:
This gives me a little pause because I know we discussed whether to rename `folder_link` to `folderLink` in some context and were worried about collisions. I don't remember the exact circumstances though. Does that sound familiar at all?

Reply:
Right right, that makes sense! Thanks Dan. And yes, it does.